src/common/users.ts

```diff
@@ -304,7 +304,7 @@ export const getUserReadingTags = (
   { userId, dateRange: { start, end }, limit = 8 }: ReadingDaysArgs,
 ): Promise<TagsReadingStatus[]> => {
   return con.query(
-    `
+    `--sql
       WITH filtered_view AS (
         SELECT
           v.*,
```

Comment:
by adding `--sql` to any sql string literal we get auto sql highlight

Reply:
<img width="605" alt="image" src="https://github.com/user-attachments/assets/880cef0d-68a7-415b-b06b-eaffbdf2a7e3">